sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpChannelProcessor.java

```diff
@@ -54,13 +52,15 @@
 
     public AmqpChannelProcessor(String fullyQualifiedNamespace, String entityPath,
         Function<T, Flux<AmqpEndpointState>> endpointStatesFunction, AmqpRetryPolicy retryPolicy, ClientLogger logger) {
-        this.fullyQualifiedNamespace = Objects
-            .requireNonNull(fullyQualifiedNamespace, "'fullyQualifiedNamespace' cannot be null.");
-        this.entityPath = Objects.requireNonNull(entityPath, "'entityPath' cannot be null.");
         this.endpointStatesFunction = Objects.requireNonNull(endpointStatesFunction,
             "'endpointStates' cannot be null.");
         this.retryPolicy = Objects.requireNonNull(retryPolicy, "'retryPolicy' cannot be null.");
-        this.logger = Objects.requireNonNull(logger, "'logger' cannot be null.");
+
+        /*Map<String, Object> loggingContext = new HashMap<>();
+        loggingContext.put(FULLY_QUALIFIED_NAMESPACE_KEY, Objects.requireNonNull(fullyQualifiedNamespace, "'fullyQualifiedNamespace' cannot be null."));
+        loggingContext.put(ENTITY_PATH_KEY, Objects.requireNonNull(entityPath, "'entityPath' cannot be null."));*/
```

Comment:
`fullyQualifiedNamespace`  (which is actually `connectionId`  https://github.com/Azure/azure-sdk-for-java/blob/0257cc9b46460277e53dd42442a13a9afc07ae6a/sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java#L398) is not used in this class anymore - it was only used for logging and checked in `this.errorContext = new AmqpErrorContext(fullyQualifiedNamespace);` a few lines later. 

`entityPath` is not used at all anymore (and I'm not sure if deleting it is a good idea since it might break with invalid core-amqp and EH/SB version combinations)